pkg/templatelib/lib.go

```diff
@@ -68,55 +69,33 @@ func stringsModifierActionFactory(a func(string, string) string) func([]string,
 	}
 }
 
-var FuncMap = template.FuncMap{
```

Comment:
I guess I could get fancy with github.com/imdario/mergo to avoid changing the signature of `templatelib.FuncMap`, but I'm not aware of any actual users of it, so I'm not sure it's worth going to the trouble? :innocent:

(I'd love to convert our uses of functions like `trimSuffixes`, `trimPrefixes`, `join`, and `getenv` to be compatible with their Sprig counterparts so we could deprecate/remove our custom versions too, which shouldn't be *too* much work.)

Reply:
https://pkg.go.dev/github.com/docker-library/go-dockerlibrary@v0.0.0-20200821205225-669fbe5c1d52/pkg/templatelib?tab=importedby

https://pkg.go.dev/github.com/docker-library/bashbrew@v0.1.1/pkg/templatelib?tab=importedby

:eyes: